test/cppsuite/test_harness/workload/random_generator.h

```diff
@@ -61,7 +61,14 @@ class random_generator {
 
     /* Generate a random integer between min and max. */
     int64_t
-    generate_integer(int64_t min, int64_t max)
+    generate_signed_integer(int64_t min, int64_t max)
```

Comment:
Can we create a template for the generate_xx methods ?

Reply:
Yeah I'll add a template here.

Reply:
72d86a133